main.md

```diff
@@ -1036,6 +1036,25 @@ Use of the `dpop_jkt` authorization request parameter is OPTIONAL.
 Note that the `dpop_jkt` authorization request parameter MAY also be used
 in combination with PKCE [@RFC7636].
 
+# DPoP with Pushed Authorization Requests
+
+When Pushed Authorization Requests  (PAR, [@RFC9126]) are used in conjunction with DPoP, there are two ways in which the DPoP key can be communicated in the PAR request: 
+
+ * The `dpop_jkt` parameter can be used as described above to bind the issued
+   authorization code to a specific key. In this case, `dpop_jkt` MUST be included alongside other authorization request parameters in the POST body of the PAR request. 
+ * Alternatively, the `DPoP` header can be added to the PAR request. In this
+   case, the authorization server MUST check the provided DPoP proof JWT as
+   defined in (#checking). It MUST further behave as if the contained public key's
+   thumbprint was provided using `dpop_jkt`, i.e., reject the subsequent token
+   request unless a DPoP proof for the same key is provided. This can help to
+   simplify the implementation of the client, as it can "blindly" attach the
+   `DPoP` header to all requests to the authorization server regardless of the
+   type of request. Additionally, it provides a stronger binding, as the `DPoP`
+   header contains a proof of possession of the private key.
+
+Both mechanisms MUST be supported by an authorization server that supports PAR and DPoP. If both mechanisms are used at the same time, the authorization server MUST reject the request if the JWK Thumbprint in `dpop_jkt` does not match the public key in the `DPoP` header. The authorization server MUST NOT enforce the usage of a nonce using the provisions of (#ASNonce) if no `DPoP` header was supplied by the client in the PAR request.
```

Comment:
> The authorization server MUST NOT enforce the usage of a nonce using the provisions of (#ASNonce) if no `DPoP` header was supplied by the client in the PAR request.

Isn't this supposed to say the AS must not enforce the usage of a nonce ... if no `nonce` claim was supplied by the client in the DPoP proof sent with the PAR request?

Likewise, should a previously received `nonce` be provided by a client in the proof, the AS is still expected to verify it is valid, and respond accordingly if it is not, right?

Reply:
So what I thought this was saying is that enforcing nonce use on the PAR endpoint shouldn't be a thing.

I see what you mean now but it's just very confusing the way it's written. It's meant to say the client MAY also send dpop_jkt but omit the DPoP proof during PAR. The confusing part is you're linking to AS nonce enforcement provisions that are hinged on DPoP Proof presence in the first place...

Reply:
The AS could send a 400 Bad Request with use_dpop_nonce, even if there's not DPoP header in the PAR request... that's what I want to prevent.

Reply:
The AS could send a 400 bad request with invalid_request because the DPoP header is missing too. I don't think we need to enumerate all possible responses so long as we instead say that a PAR request with dpop_jkt parameter without DPoP Header must be accepted as valid (barring other discussions around always_uses_dpop).

Reply:
I agree with @panva that a PAR request with dpop_jkt parameter without DPoP Header must be accepted as valid, given that the DPoP header can be subsequently sent in the token request.  Can we add something along those lines to the PR?

Reply:
That part is already clear from the text, I suppose. The question was around the last sentence, whether we should explicitly forbid that an AS insists on a nonce on the PAR endpoint. I removed the sentence for now, we can discuss this later on if the topic comes up again.